src/script/components/MessagesList/Message/FailedToAddUsersMessage.tsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React, {useMemo, useState} from 'react';
+
+import {container} from 'tsyringe';
+import {groupBy} from 'underscore';
+
+import {Button, ButtonVariant, COLOR_V2, Link, LinkVariant} from '@wireapp/react-ui-kit';
+
+import {Icon} from 'Components/Icon';
+import {Config} from 'src/script/Config';
+import {User} from 'src/script/entity/User';
+import {UserState} from 'src/script/user/UserState';
+import {useKoSubscribableChildren} from 'Util/ComponentUtil';
+import {t} from 'Util/LocalizerUtil';
+import {matchQualifiedIds} from 'Util/QualifiedId';
+
+import {warning} from './ContentMessage/Warnings/Warnings.styles';
+import {MessageTime} from './MessageTime';
+
+import {FailedToAddUsersMessage as FailedToAddUsersMessageEntity} from '../../../entity/message/FailedToAddUsersMessage';
+
+export interface FailedToAddUsersMessageProps {
+  message: FailedToAddUsersMessageEntity;
+  userState?: UserState;
+}
+
+const FailedToAddUsersMessage: React.FC<FailedToAddUsersMessageProps> = ({
+  message,
+  userState = container.resolve(UserState),
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const {timestamp} = useKoSubscribableChildren(message, ['timestamp']);
+  const config = Config.getConfig();
+
+  const {users: allUsers} = useKoSubscribableChildren(userState, ['users']);
+
+  const [users, total, groupedUsers] = useMemo(() => {
+    const users: User[] = message.qualifiedIds.reduce<User[]>((previous, current) => {
+      const foundUser = allUsers.find(user => matchQualifiedIds(current, user.qualifiedId));
+      if (foundUser) {
+        return [...previous, foundUser];
+      }
+      return previous;
+    }, []);
+    const groupedUsers = groupBy(users, user => user.domain);
+    const total = users.length;
+    return [users, total, groupedUsers];
+  }, [allUsers, message.qualifiedIds]);
+
+  if (users.length === 0) {
+    return null;
+  }
+
+  const learnMore = (
+    <>
+      {' '}
+      <Link
+        targetBlank
+        variant={LinkVariant.PRIMARY}
+        href={config.URL.SUPPORT.USER_ADD_FAILURE}
+        data-uie-name="go-contact-support"
+        style={{
+          color: COLOR_V2.BLUE_LIGHT_500,
```

Comment:
Did you check if the color was blue or accent-color for the link? it's not obvious on the mock.

If it's blue, we can use `var(--blue-500)` , it should have the correct value in light and dark mode

Reply:
https://github.com/wireapp/wire-webapp/pull/15187/commits/939ddea6778a66f5256c5134477a96a9349eb21f